code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -219,12 +219,15 @@
 	return
 
 /datum/species/ipc/spec_life(mob/living/carbon/human/H)
-	. = ..()
-	if(H.health <= UNCONSCIOUS && H.stat != DEAD) // So they die eventually instead of being stuck in crit limbo.
-		H.adjustFireLoss(6) // After bodypart_robotic resistance this is ~2/second
+	H.setOxyLoss(0)
+	H.losebreath = 0
```

Comment:
why? They already cannot take oxy damage whatsoever and this just adds more proc overhead to Life() for no reason

Reply:
this is from the spec_life proc, which i overrode. This calls for every species with nobreath()